### PR TITLE
Updated mannequin stand and added door scanners

### PR DESCRIPTION
**Pull request by LinkTheRetroGameArtist**
updated stand and new asset

### CODE REVIEW

**Comment by troylusty** [comment]
@LinkTheRetroGameArtist What other asset have you included within this commit alongside the updated stand?

**Comment by LinkTheRetroGameArtist** [comment]
> @LinkTheRetroGameArtist What other asset have you included within this commit alongside the updated stand?

Door scanners aka Scanner_Sm in the bianary files and door scanner said in title and files changed lol yellow is edit , green is new new file

File: Content/Assets/scanner.uasset
Comment (troylusty) [inline Content/Assets/scanner.uasset:1]: This asset appears to be a duplicate of another within this commit.

Comment (LinkTheRetroGameArtist) [reply Content/Assets/scanner.uasset:1]: its caused i changed the name there only one file on my side i cant change that the scanner and scanner sm are the same assest in github eyes yes

Comment (troylusty) [reply Content/Assets/scanner.uasset:1]: I'll get rid of it when I get home

**Review by troylusty — APPROVED** [review]
Removed duplicate uasset file

**Pull request merged**